srtcore/group.cpp

```diff
@@ -892,6 +892,12 @@ void CUDTGroup::close()
                 HLOGC(smlog.Debug, log << "group/close: IPE(NF): group member @" << ig->id << " already deleted");
                 continue;
             }
+
+            // XXX This is not true in case of non-managed groups, which
+            // only collect sockets, but also non-managed groups should not
+            // use common group buffering and tsbpd.
```

Comment:
What's not true? Could you please clarify the comment?

Reply:
Right. Likely missed some earlier part that was only valid for common group receiver.